packages/blade/src/components/Dropdown/DropdownOverlay.web.tsx

```diff
@@ -163,7 +167,20 @@ const _DropdownOverlay = ({ children, testID }: DropdownOverlayProps): JSX.Eleme
   return (
     // eslint-disable-next-line @typescript-eslint/no-explicit-any
     <BaseBox position="relative" ref={refs.setFloating as any}>
-      <StyledDropdownOverlay
+      {isOpen ? (
+        <BaseBox
+          position={'fixed' as never}
+          top="spacing.0"
+          left="spacing.0"
+          height="100%"
+          width="100%"
+          // eslint-disable-next-line @typescript-eslint/ban-ts-comment, @typescript-eslint/prefer-ts-expect-error
+          // @ts-ignore: throws TS error that onClick doesn't exist in native even though its just web file
```

Comment:
Why is this happening? It shouldn't check web file unless its being imported in native somewhere. Can you check it?

Reply:
aha yes I was importing prop types from web file. moved it `types.ts` file now.